README.md

```diff
@@ -51,7 +51,8 @@ To configure the application before installation and usage (see the next section
     ----- | -----
     `LOG_LEVEL` | The minimum level for log messages that will appear in output. `INFO` or `DEBUG` is recommended for most use cases; see [Python's logging module](https://docs.python.org/3/library/logging.html).
     `CANVAS_ACCOUNT_ID` | The Canvas instance root account ID number associated with the courses for which data will be collected.
-    `CANVAS_TERM_ID` | The Canvas instance term ID number that will be used to limit the query for Canvas courses.
+    `CANVAS_TERM_ID` | The Canvas instance term ID number that will be used to limit the query for Canvas courses. Set to 0 to just use ADD_COURSE_IDS.
+    `ADD_COURSE_IDS` | Additional CourseID's to retrieve. Duplicates found in CANVAS_TERM_ID (if defined) are not currently removed.
```

Comment:
It looks a little awkward, but could we figure out how to drop duplicates? Maybe some kind of mapping function that checks the course ID on the course object? Seems like something good to do in preparation for database insertion, though you could just drop duplicates later. It'll be a couple less requests though! 

I think it might streamline the code a little if you did that first, and then iterated over the unique course objects all at once.

Reply:
Also feeling like this could be incorporated in other parts of the codebase, but that can definitely wait.

Reply:
Yeah, I think we *could* do it. We'd either have to skip those courses defined when going through the term for loop or keep track of these and not process them afterward. I'm feeling like real world usage isn't that you'll specify courses and a term, that it would be one or another. There's no reason to add courses that are part of the term as it will already be processed. We can see if this is really an issue. :)

Reply:
And yeah, I'd like to refactor this code (again) to use the threading calls too so that may end up making this a lot more straightforward to do.